frontend/src/components/connection/RequestsPage.js

```diff
@@ -62,20 +62,20 @@ function RequestsPage() {
         });
     });
   }, [Users]);
-//This function will add the connection to the database whenver a user accepts the connection invitation. 
+  //This function will add the connection to the database whenver a user accepts the connection invitation.
   function handleConnect(userId) {
     const connectionRef = collection(db, "connection");
     const authdoc = doc(connectionRef, currentId);
     const acceptedoc = doc(connectionRef, userId);
-    const array = []
+    const array = [];
```

Comment:
The name of this variable could use some work. Labelling it as "array" doesn't provide much information.